### PR TITLE
Update server failover documentation.

### PR DESCRIPTION
**Pull request by bbilbrey-aveva**
# Changes
- Describe the read-only redundant server set cache facet in the principles of operation.
- Adds an administration page to describe the process redundant server set endpoint. 
- Adds `Attempting Failover` as a recovery interval triggering event

### CODE REVIEW

File: content/main/shared-content/administration/server-redundancy-set.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+uid: RetrieveRedundancySet
+---
+
+# Retrieve Redundant Server Set
+
+The AVEVA Opc Ua Adapter provides a rest endpoint for querying the current redundant server set the adapter is using for server failover.
```
Comment (aveva-jhwang) [inline content/main/shared-content/administration/server-redundancy-set.md:7]: Could we update it to capitalize "Opc Ua" to "OPC UA"?

File: content/main/shared-content/administration/server-redundancy-set.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Complete the following steps to retrieve the redundant server set of an AVEVA Opc Ua Adapter:
```
Comment (aveva-jhwang) [inline content/main/shared-content/administration/server-redundancy-set.md:9]: Same comment about capitalization as earlier.

File: content/main/shared-content/administration/server-redundancy-set.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+   ```bash
+   curl -X GET "http://localhost:5590/api/v1/configuration/<ComponentId>/RedundantServerSet"
+   ```
+
+   Example result:
+
+    ```json
+    {
+        "ServerAddresses": [
+            "opc.tcp://Server1"
```
Comment (aveva-jhwang) [inline content/main/shared-content/administration/server-redundancy-set.md:29]: Is the server list not comma delimited? Seems odd seeing it without some kind of visible delimiter.

File: content/main/shared-content/configuration/history-recovery/automatic-history-recovery.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ uid: AutomaticHistoryRecovery
...
+For automatic history recovery, the adapter tracks changes to the **DeviceStatus** of each component. When the **DeviceStatus** changes to `DeviceInError` or `Shutdown` or `Attempting Failover`, the adapter starts a new [History recovery interval](#history-recovery-intervals). The **DeviceStatus** changes to `Good` when the issue resolves or if the adapter restarts and then the adapter closes any current intervals for that component. The adapter tracks these intervals for each component and, when **DeviceStatus** has a value of `Good`, it performs history recovery for these intervals from oldest entry to newest. For more information, see also [Device status](xref:DeviceStatus).
```
Comment (aveva-jhwang) [inline content/main/shared-content/configuration/history-recovery/automatic-history-recovery.md:9]: Remove "or" between 'DeviceInError' and 'Shutdown'. 
The statement "when the issue resolves" sounds awkward to me. Maybe something like "when the issue is resolved"?
For the statement "or if the adapter restarts and then the adapter closes any current intervals for that component" is a bit wordy. Suggestion: "or if the adapter restarts and closes any current intervals for that component".
Should it be "the oldest entry to the newest"?

File: content/overview/principles-of-operation.md [hunk 5]
```diff
@@ -164,9 +164,9 @@ In Hot server failover, a failover occurs if the adapter loses connection to the
 
 ### Redundancy Server Set Cache
 
-On startup, when the adapter reads the Server URI array, it will store the results in a file on disk in the following locations:
+On startup, when the adapter connects to the initial primary server and reads the Server URI array, it will store the results in a json file at:
```
Comment (aveva-jhwang) [inline content/overview/principles-of-operation.md:167]: Suggestion: rather than "in a json file at", say "in a json file in the following directories"

**Review by aveva-jhwang — COMMENTED** [review]
I added some comments and suggestions. Let me know if you have any questions regarding them.

**Comment by aveva-jhwang** [comment]
I just realized that Brandon already addressed a couple of grammatical items that I pointed out in a different PR (#97).

File: content/main/shared-content/administration/server-redundancy-set.md (hunk 1 above)
Comment (bbilbrey-aveva) [reply content/main/shared-content/administration/server-redundancy-set.md:7]: This was resolved in a subsequent PR.

File: content/main/shared-content/administration/server-redundancy-set.md (hunk 2 above)
Comment (bbilbrey-aveva) [reply content/main/shared-content/administration/server-redundancy-set.md:9]: Resolved in a subsequent PR.

File: content/main/shared-content/configuration/history-recovery/automatic-history-recovery.md (hunk 4 above)
Comment (bbilbrey-aveva) [reply content/main/shared-content/configuration/history-recovery/automatic-history-recovery.md:9]: This was resolved in a subsequent PR.
